meta-schema.graphql

```diff
@@ -0,0 +1,115 @@
+# Base types
+type Party {
+	id:ID!
+	idType:String! #PartyId, IPI, ISNI, MBID
```

Comment:
Should these not be individual fields? For example, if a single artist has all of these identifiers, they should be returned as a single `Party` rather than 4 different ones right?

Reply:
Absolutely, oversight on my behalf.